server.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+'use strict';
+
+const http = require('http');
+const getHandler = require('./getHandler');
+const postHandler = require('./postHandler');
+const putHandler = require('./putHandler');
+
+
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  console.log(req.method, req.url, req.httpVersion);
+
+  switch(req.method) {
```

Comment:
This is really good. This makes reading code so much easier. You're actually the only one that did this too.